merchant/src/catalog/backend.rs

```diff
@@ -642,12 +646,18 @@ impl CatalogService for CatalogSQLService {
             .pool
             .acquire()
             .await
-            .map_err(|_| CatalogError::DatabaseError)?;
+            .map_err(|err| {
+                println!("{:?}", err);
```

Comment:
✂️ ... in a separate PR we could introduce the `log` crate replacing all occurences of `println!`